File: EExpress/EExpress/Scripts/EExpress/controllers/siteDomainController.js
app.controller("siteDomainController", function ($scope, siteDomainService) {

    $scope.m = {};
    $scope.message = "";

    $scope.resetForm = function () {
        $scope.m = {};
    }

    $scope.getOrigin = function () {
        var request = siteDomainService.GetOrigin();

        request.then(function (response) {
            $scope.listOrigin = response.data;
        });
    }

    $scope.getSiteDomain = function () {
        var request = siteDomainService.GetSiteDomain();

        request.then(function (response) {
            $scope.listSiteDomain = response.data;
        }, function (response) {
            $scope.message = response.statusText;
        });
    }

    $scope.onSelectChange = function () {
        var request = siteDomainService.GetOriginBySiteName($scope.m.nmd);

        request.then(function (response) {
            var origin = response.data;

            $scope.m = {
                my_domain: origin.kdkota,
                nmd: origin.nm
            };
        }, function (response) {
            $scope.message = response.statusText;
        });
    }

    $scope.addEditSiteDomain = function () {
        var siteDomain = {
            my_domain: $scope.m.my_domain,
            nmd: $scope.m.nmd
        };

        var request = siteDomainService.AddEditSiteDomain(siteDomain);

        request.then(function (response) {
            $scope.resetForm();
            $scope.getOrigin();

            $scope.message = response.data;
        }, function (response) {
            $scope.message = response.statusText;
        });
    }

    $scope.getOrigin();
    $scope.getSiteDomain();
})

File: EExpress/EExpress/Controllers/CourierCargo/MasterData/SiteDomainController.cs
using EExpress.Models;
using EExpress.Models.DbHandlers;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace EExpress.Controllers.CourierCargo.MasterData
{
    public class SiteDomainController : Controller
    {
        SiteDomainDbHandler db = new SiteDomainDbHandler();

        // GET: SiteDomain
        public ActionResult Index()
        {
            return View();
        }

        public JsonResult GetSiteDomain()
        {
            var listSiteDomain = db.GetSiteDomain();

            return Json(listSiteDomain, JsonRequestBehavior.AllowGet);
        }

        public JsonResult GetOrigin()
        {
            var listOrigin = db.GetOrigin();

            return Json(listOrigin, JsonRequestBehavior.AllowGet);
        }

        public JsonResult GetOriginBySiteName(string siteName)
        {
            var origin = db.GetOriginBySiteName(siteName);

            return Json(origin, JsonRequestBehavior.AllowGet);
        }

        [HttpPost]
        public JsonResult AddEditSiteDomain(SiteDomain siteDomain)
        {
            if (ModelState.IsValid)
            {
                db.AddEditSiteDomain(siteDomain);
            }

            string status = !string.IsNullOrEmpty(siteDomain.my_domain)  ? "updated" : "saved";
            string message = $"Site / Domain has been {status} successfully";
            return Json(message, JsonRequestBehavior.AllowGet);
        }
    }
}